pkg/util/logz/logz.go

```diff
@@ -0,0 +1,45 @@
+package logz
```

Comment:
why `-z` in the package name, tryin' to be cool brah? :)

Reply:
log + zap = logz? If you have a better name for it...?